_buy/fips201apl-pacs.md

```diff
@@ -46,66 +46,19 @@ Visit the [Buy Page](../) to view FICAM products, services and purchasing guidan
-              <ul class="usa-unstyled-list">
-                <ol type = "1">
-                {% if guide.reader1 %}
-                  <li><a href="{{ guide.reader1url | prepend: site.baseurl }}" target="_blank">{{ guide.reader1 }}</a></li>
-                {% endif %}
-                {% if guide.reader2 %}
-                  <li><a href="{{ guide.reader2url | prepend: site.baseurl }}" target="_blank">{{ guide.reader2 }}</a></li>
-                {% endif %}
-                {% if guide.reader3 %}
-                  <li><a href="{{ guide.reader3url | prepend: site.baseurl }}" target="_blank">{{ guide.reader3 }}</a></li>
-                {% endif %}
-                {% if guide.reader4 %}
-                  <li><a href="{{ guide.reader4url | prepend: site.baseurl }}" target="_blank">{{ guide.reader4 }}</a></li>
-                {% endif %}
-                {% if guide.reader5 %}
-                  <li><a href="{{ guide.reader5url | prepend: site.baseurl }}" target="_blank">{{ guide.reader5 }}</a></li>
-                {% endif %}
-                {% if guide.reader6 %}
-                  <li><a href="{{ guide.reader6url | prepend: site.baseurl }}" target="_blank">{{ guide.reader6 }}</a></li>
-                {% endif %}
-                {% if guide.reader7 %}
-                  <li><a href="{{ guide.reader7url | prepend: site.baseurl }}" target="_blank">{{ guide.reader7 }}</a></li>
-                {% endif %}
-                {% if guide.reader8 %}
-                  <li><a href="{{ guide.reader8url | prepend: site.baseurl }}" target="_blank">{{ guide.reader8 }}</a></li>
-                {% endif %}
-                {% if guide.reader9 %}
-                  <li><a href="{{ guide.reader9url | prepend: site.baseurl }}" target="_blank">{{ guide.reader9 }}</a></li>
-                {% endif %}
-                {% if guide.reader10 %}
-                  <li><a href="{{ guide.reader5url | prepend: site.baseurl }}" target="_blank">{{ guide.reader10 }}</a></li>
-                {% endif %}
-                {% if guide.reader11 %}
-                  <li><a href="{{ guide.reader6url | prepend: site.baseurl }}" target="_blank">{{ guide.reader11 }}</a></li>
-                {% endif %}
-                {% if guide.reader12 %}
-                  <li><a href="{{ guide.reader7url | prepend: site.baseurl }}" target="_blank">{{ guide.reader12 }}</a></li>
-                {% endif %}
-                {% if guide.reader13 %}
-                  <li><a href="{{ guide.reader8url | prepend: site.baseurl }}" target="_blank">{{ guide.reader13 }}</a></li>
-                {% endif %}
-                {% if guide.reader14 %}
-                  <li><a href="{{ guide.reader9url | prepend: site.baseurl }}" target="_blank">{{ guide.reader14 }}</a></li>
-                {% endif %}
-                </ol>
-              </ul>
-            </td>
+             <td headers="pacs-table-heading-{{ category | slugify }} pacs-table-heading-valapl">{{ guide.valapl }}</td>
```

Comment:
@maxwellfunk what happens with the card reader pdfs? Do they stay on the site or remove them? They should be linked somewhere.

Reply:
Currently developing a new system for linking to the card reader APLs... branch link below:
https://github.com/GSA/idmanagement.gov/tree/Reader_list_addition